edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -124,7 +124,7 @@ where
                     let image_id = match image_name_to_id.get(config.image()) {
                         Some(imageid) => imageid,
                         None => {
-                            log::warn!("{} was not added to image garbage collection list and may will not be garbage collected", image);
+                            log::warn!("{} was not added to image garbage collection list (because the image id could not be retrieved) and will not be garbage collected", image);
```

Comment:
Curious in what case this message comes up. Recommend aplittting into multiple sentence not parenths.